dissect/ntfs/mft.py

```diff
@@ -46,6 +46,9 @@ class Mft:
     def __init__(self, fh: BinaryIO, ntfs: Optional[NTFS] = None):
         self.fh = fh
         self.ntfs = ntfs
+        self.record_size = self.ntfs._record_size if self.ntfs else DEFAULT_RECORD_SIZE
+        self.mft_size = self.get(FILE_NUMBER_MFT).size()
```

Comment:
This may not be correct at the time of `Mft` class initialisation. See https://github.com/fox-it/dissect.ntfs/blob/d6c039c6b4959a5d1ea5b012673813d2ce91693a/dissect/ntfs/ntfs.py#L89-L114 for the reasoning why.

Best to leave this where it was IMO.